docs/en/building/basic-usage.md

```diff
@@ -0,0 +1,118 @@
+# Basic Usage
+
+## Creating a Dojo application
+
+The first step for building a Dojo application is to install the CLI
+
+> Command line
+
+```bash
+npm install -g @dojo/cli @dojo/cli-create-app
+```
+
+This command installs the Dojo command-line tool (`@dojo/cli`) and the Dojo CLI module to create an application (`@dojo/cli-create-app`). There are a number of CLI modules that add commands and functionality to the dojo command-line.
+
+Now create an application using the dojo command.
+
+> Command line
+
+```bash
+dojo create app --name my-first-app
+```
+
+This command will generate a basic project structure in the `my-first-app` directory. This project will include a `package.json` file, an example Dojo application, and all of the project dependencies will be pre-installed. It will include two additional Dojo CLI modules. [@dojo/cli-build-app](https://github.com/dojo/cli-build-app) and [@dojo/cli-test-intern](https://github.com/dojo/cli-test-intern).
+
+## Building
+
+There are several types of build targets or `mode`s that Dojo's CLI tool supports. All of these modes can be seen in the various `package.json` scripts generated by `dojo create app`.
+
+Run the following to create an optimized build ready for production.
+
+> Command line
+
+```bash
+npm run build
```

Comment:
Can we use the "real" dojo commands through out, because unless users start with the cli template the changes are they wont have the same npm scripts.